eng/templates/jobs/ci-unit-tests.yml

```diff
@@ -35,19 +35,22 @@ jobs:
         displayName: 'Install dependencies'
         condition: and(eq(variables.isSdkRelease, false), eq(variables.isExtensionsRelease, false), eq(variables['USETESTPYTHONSDK'], false), eq(variables['USETESTPYTHONEXTENSIONS'], false))
       - bash: |
-          echo "Python version from variable: $PYTHON_VERSION"
-          if (( $(echo "$PYTHON_VERSION < 3.13" | bc -l) )); then
-              echo "Running unittests (Python < 3.13)..."
+          PY_VER="$(PYTHON_VERSION)"
+          echo "Python version: $PY_VER"
+      
+          # Extract minor version as integers
+          PY_MINOR="${PY_VER#*.}"
+      
+          if [ "$PY_MINOR" -ge 13 ]  then
```

Comment:
NIT: Consider having a file that contains the supported version of python and used at both ci-unit-tests and ci-e2e-tests.yml file.